intermediate_notebooks/E2E/taxi/NYCTaxi-E2E.ipynb

```diff
@@ -339,9 +695,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 1.26 s, sys: 104 ms, total: 1.36 s\n",
+      "Wall time: 12.2 s\n"
```

Comment:
The intent here is to show ETL time against the full dataset, as opposed to just the months I think you were using for testing your changes.